frontend/src/config/mozambique/layers.json

```diff
@@ -2239,5 +2239,38 @@
       }
     ],
     "legend_text": "Projected 2022 population data. Source: INE"
+  },
+  "acled_incidents": {
+    "title": "ACLED incidents",
+    "loader": "acled",
+    "type": "point_data",
+    "data": "http://localhost/acled",
```

Comment:
Or do you mean to point to the API?

Reply:
I think you need to use a similar approach to what we do for the stats API. If that's helpful, you could review this PR https://github.com/WFP-VAM/prism-app/pull/559 so that we can get it merged before this one.

Reply:
We need to point to the API to include the acled credentials in the backend side. Then, the data returned is processed within the frontend. I don't believe #559 affects this PR. In any case, It could be addressed as a separate issue.